sky/shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -190,6 +193,12 @@ float getDevicePixelRatio() {
     }
 
     public void destroy() {
+        mPlatformServiceProviderBinding.unbind().close();
+        mPlatformServiceProvider.destroy();
```

Comment:
Do we need to null-check these?  Is it possible that we're told to destroy before we've created them?